services/QuillLMS/app/models/students_classrooms.rb

```diff
@@ -29,17 +29,26 @@ class StudentsClassrooms < ApplicationRecord
     if: proc { |sc| sc.archived? && sc.student && sc.classroom },
     unless: :skip_archive_student_associations
 
+  # after_save :save_user_pack_sequence_items, if: [:saved_change_to_visible?]
```

Comment:
Why is this commented out?

Reply:
Good eye.  That is a temporary comment out while I figure out a deadlock in testing.